src/commands/build/mod.rs

```diff
@@ -9,7 +9,14 @@ use std::process::Command;
 
 use crate::emoji;
 
-pub fn build(cache: &Cache, project_type: &ProjectType) -> Result<(), failure::Error> {
+pub fn build(cache: &Cache, project: &Project) -> Result<(), failure::Error> {
+    let project_type = &project.project_type;
+    let webpack_config_path = PathBuf::from(
+        &project
+            .webpack_config
+            .clone()
```

Comment:
ideally we can avoid clones but sometimes it's not ideal for code readability :) that being said i think this is fine for now, we should just audit these generally

Reply:
Yeah I didn't really like it, just what the compiler recommended. I'm still a little iffy on how Rust handles borrowing + what is idiomatic.